mne/viz/_3d.py

```diff
@@ -1718,6 +1719,310 @@ def _get_ps_kwargs(initial_time, require='0.6'):
+                              ax_x.lines[0].get_xdata()[0],
+                              ax_x.lines[1].get_ydata()[0])
+            ax_x.clear()
+            ax_y.clear()
+            ax_z.clear()
+            params.update({'img_idx': index_img(img, idx)})
+            params.update({'title': 'Activation (t=%.3f s.)'
+                          % params['stc'].times[idx]})
+            plot_map_callback(
+                params['img_idx'], title='',
+                cut_coords=cut_coords)
+
+        if event.inaxes in [ax_x, ax_y, ax_z]:
+            if mode == 'stat_map':
+                cut_coords = _get_cut_coords_stat_map(event, params)
+            elif mode == 'glass_brain':
+                cut_coords = _get_cut_coords_glass_brain(event, params)
+
+            x, y, z = cut_coords
+            ax_x.clear()
+            ax_y.clear()
+            ax_z.clear()
+            plot_map_callback(params['img_idx'], title='',
+                              cut_coords=cut_coords)
+            loc_idx = _cut_coords_to_idx(cut_coords, params['img_idx'])
+
+            if mode == 'glass_brain':
+                _glass_brain_crosshairs(params, *cut_coords)
+            if loc_idx is not None:
+                ax_time.lines[0].set_ydata(stc.data[loc_idx].T)
+            else:
+                ax_time.lines[0].set_ydata(0.)
+        params['fig'].canvas.draw()
+
+    if bg_img is None:
+        subjects_dir = get_subjects_dir(subjects_dir=subjects_dir,
+                                        raise_error=True)
+        subject = _check_subject(stc.subject, subject, True)
+        t1_fname = op.join(subjects_dir, subject, 'mri', 'T1.mgz')
+        bg_img = nib.load(t1_fname)
+
+    bg_img_param = bg_img
+    if mode == 'glass_brain':
+        bg_img_param = None
+
+    img = stc.as_volume(src, mri_resolution=False)
+
+    vmax = np.abs(stc.data).max()
+    loc_idx, idx = np.unravel_index(np.abs(stc.data).argmax(),
+                                    stc.data.shape)
+    img_idx = index_img(img, idx)
+    x, y, z = np.unravel_index(stc.vertices[loc_idx], img_idx.shape,
+                               order='F')
+    cut_coords = apply_trans(img.affine, (x, y, z))
+
+    # Plot initial figure
+    fig = plt.figure()
+
+    vmin = -vmax
+    if np.all(stc.data > 0):
+        vmin = 0.
+    ax_time = fig.add_axes([0.09, 0.1, 0.9, 0.4], ylim=(vmin, vmax))
+    ax_time.plot(stc.times, stc.data[loc_idx].T, color='k')
+    ax_time.set(xlim=stc.times[[0, -1]])
+    lx = ax_time.axvline(stc.times[idx], color='g')
+    plt.xlabel('Time (ms)')
+    plt.ylabel('Activation')
+
+    ctrl_pts, colormap, scale_pts, _ = _limits_to_control_points(
+        clim, stc.data, colormap, transparent, fmt='matplotlib')
+    if np.array_equal(ctrl_pts, scale_pts):  # set eq above iff one-sided
+        # there is a bug in nilearn where this messes w/transparency
+        # Need to double the colormap
+        if (ctrl_pts < 0).any():
```

Comment:
actually this won't be negative even if the data is signed

Reply:
the user can pass negative control points, I'll clarify the message and add a test